src/Integration.Vsix/CFamily/CFamilyHelper.cs

```diff
@@ -196,7 +195,7 @@ private static void SaveFileConfig(FileConfig fileConfig, ILogger logger)
             runner.Execute(args);
         }
   
-        internal static FileConfig TryGetConfig(ILogger logger, ProjectItem projectItem, string absoluteFilePath)
+        internal static FileConfig TryGetConfig(ILogger logger, ProjectItem projectItem, string absoluteFilePath, IAnalyzerOptions analyzerOptions)
```

Comment:
@duncanp-sonar I realized that I might be logging the fileConfig a bit too late. In the previous version, it was possible that Request was null due to some other failure, and so we wouldn't have reached a point where we log the file config.
So now I log it directly after we get it.